mmv1/products/redis/Cluster.yaml

```diff
@@ -108,6 +108,9 @@ timeouts:
   update_minutes: 120
   delete_minutes: 30
 autogen_async: true
+datasource:
+  generate: true
+  exclude_test: true
```

Comment:
It looks like redis still isn't quite equivalent on generation - we're losing the call to `SetDataSourceLabels`. Let's just undo the generation (and put back the handwritten file) for now so this PR can focus on the test generation changes.

```suggestion
```